src/_helpers/utils.js

```diff
@@ -1,7 +1,7 @@
 import moment from 'moment';
 
 const handleResponse = resp => {
```

Comment:
I think all error handling can be moved into axios interceptor (`src/index.js`)? Perhaps it's time to deprecate this function. Lmk what you think.

Reply:
That does sound good tbf, I didn't know that axios interceptor existed. However, I will not do that in this pull request